src/js/apps/globals/nav_app.js

```diff
@@ -53,60 +65,80 @@ const appNavMenu = new Backbone.Collection([
       icon: 'users-gear',
     },
   },
-  {
-    onSelect() {
-      window.open('https://help.roundingwell.com/');
-    },
-    text: i18n.mainNav.help,
-    isExternalLink: true,
-    icon: {
-      type: 'far',
-      icon: 'life-ring',
-    },
-  },
-  {
-    onSelect() {
-      Radio.request('auth', 'logout');
-    },
-    hasDivider: true,
-    text: i18n.mainNav.signOut,
-    icon: {
-      type: 'fas',
-      icon: 'right-from-bracket',
-    },
-  },
 ]);
 
 const patientsAppWorkflowsNav = new Backbone.Collection([
   {
     text: i18n.patientsAppNav.ownedBy,
+    icons: [{
+      type: 'fas',
+      icon: 'list',
+    }],
     event: 'worklist',
     eventArgs: ['owned-by'],
   },
   {
     text: i18n.patientsAppNav.schedule,
+    icons: [{
+      type: 'fas',
+      icon: 'calendar-star',
+    }],
     event: 'schedule',
     eventArgs: [],
-    className: 'app-nav__spacer',
   },
   {
     text: i18n.patientsAppNav.sharedBy,
+    icons: [{
+      type: 'fas',
+      icon: 'arrow-right-arrow-left',
+    }],
     event: 'worklist',
     eventArgs: ['shared-by'],
-    className: 'app-nav__spacer',
   },
   {
     text: i18n.patientsAppNav.newPastDay,
+    icons: [
+      {
+        type: 'fas',
+        icon: 'angle-left',
+      },
+      {
+        type: 'fas',
+        icon: '1',
+      },
+    ],
```

Comment:
Some of these nav link items need to support having two icons placed together (`<1`, `<3`, `30` icons):

![Screenshot 2023-01-13 at 9 52 52 AM](https://user-images.githubusercontent.com/35355575/212997256-9d746709-cd4e-4637-a46f-135d04ca667e.png)

So I changed this collection take one or more `icons` instead of just one icon. In the view, I display them using a `{{#each}}` so that two icon elements are rendered next to each other like `<svg></svg><svg></svg>`. And if only icon is passed, it renders the same as one `<svg></svg>` element.